tests/foreman/endtoend/test_api_endtoend.py

```diff
@@ -432,6 +432,8 @@
         '/api/hostgroups/multiple_play_roles',
         '/api/hostgroups/:id/ansible_roles',
         '/api/hostgroups/:id/assign_ansible_roles',
+        '/api/hostgroups/:id/ansible_roles/:ansible_role_id',
+        '/api/hostgroups/:id/ansible_roles/:ansible_role_id',
```

Comment:
Non-blocking, this is just for readability, same can be done for hosts entity
```suggestion
        '/api/hostgroups/:id/ansible_roles',
        '/api/hostgroups/:id/ansible_roles/:ansible_role_id',
        '/api/hostgroups/:id/ansible_roles/:ansible_role_id',
        '/api/hostgroups/:id/assign_ansible_roles',
```

Reply:
@pondrejk Thanks for addressing this, as you commited a suggestion which doesn't updates host entity, can you update it manually please?

Reply:
ah I see, updated, thanks!